### ISSUE TITLE
Be able to run the project locally

### ISSUE THREAD

**Issue by Kerumen** [RecastAI/Webchat#23]
I wanted to contribute to the project but it seems not easy.

It would be nice if `npm start` spawns a simple webserver serving and `index.html` with the webchat loaded (with a random demo token).

It would be easier to contribute to this project!

**Comment by FKSI** [comment]
Agree! I used a quick `.html` file + fork a hello bot on the platform.

**Comment by majdas007** [comment]
please can you explain more by steps what to do after cloning the project?

**Comment by ya332** [comment]
If you add the script tag `<script src="https://cdn.cai.tools.sap/webchat/webchat.js" channelId="<Channel ID>" token="<token ID>" id="cai-webchat" ></script>` to the index.html, your bot should appear in your landing page.
